hive-hooks/path-conversion-hook/README.md

```diff
@@ -0,0 +1,79 @@
+# Path Conversion Hook
```

Comment:
So layout of the code is this:
```
hive-hooks/
├── path-conversion-hook
│   ├── README.md
│   ├── pom.xml
│   └── src
│       ├── main
│       │   └── java
│       │       └── com
│       │           └── expediagroup
│       │               └── apiary
│       │                   └── extensions
│       │                       └── hooks
│       │                           └── pathconversion
│       │                               ├── config
│       │                               │   └── Configuration.java
│       │                               ├── converters
│       │                               │   ├── GenericConverter.java
│       │                               │   └── PathConverter.java
│       │                               ├── filters
│       │                               │   └── ApiaryMetastoreFilter.java
│       │                               ├── models
│       │                               │   └── PathConversion.java
│       │                               └── providers
│       │                                   └── ApiaryNullAuthorizationProvider.java
│       └── test
│           ├── java
│           │   └── com
│           │       └── expediagroup
│           │           └── apiary
│           │               └── extensions
│           │                   └── hooks
│           │                       └── pathconversion
│           │                           ├── config
│           │                           │   └── ConfigurationTest.java
│           │                           ├── converters
│           │                           │   └── PathConverterTest.java
│           │                           └── filters
│           │                               └── ApiaryMetastoreFilterTest.java
│           └── resources
│               └── log4j2.xml
└── pom.xml
```

I suggest we remove: `path-conversion-hook` and move everything up to `hive-hooks` + move all classes that do not contain Path stuff in a higher package.
For example:
```
hive-hooks/
│   ├── README.md
│   ├── pom.xml
│   └── src
│       ├── main
│       │   └── java
│       │       └── com
│       │           └── expediagroup
│       │               └── apiary
│       │                   └── extensions
│       │                       └── hooks
│       │                              ├── config
│       │                              │   └── Configuration.java
│       │                              ├── converters
│       │                              │   └── GenericConverter.java
│       │                              ├── filters
│       │                              │   └── ApiaryMetastoreFilter.java
│       │                              └── providers
│       │                              │     └── ApiaryNullAuthorizationProvider.java
│       │                              └── pathconversion
│       │                                     ├── converters
│       │                                         └── PathConverter.java
│       │                                     ├── models
│       │                                         └── PathConversion.java
└── pom.xml
```

Reply:
Agreed that this is a bit hard to decide since this is our first hook and TBH might be our only one, I'm not aware of any others in the pipeline. I think it's worth going with what @patduin has suggested but not worrying too much about getting it perfect. If/when we get another hook we'd probably need to do some refactoring as it's hard to build something generic with only one use case.